src/care/LoopFuser.cpp

```diff
@@ -14,21 +14,6 @@
 #include "umpire/Allocator.hpp"
 #include "umpire/TypedAllocator.hpp"
 
-// TODO: when umpire is updated, you should get duplicate defines of this. Feel free to delete
-// with a vengeance.
-template <typename T, typename U>
```

Comment:
Thanks for getting rid of this! This should resolve the spack HIP build conflict that we were seeing.

Reply:
You're welcome. Thanks for figuring out this was an issue :)